src/templates/templateHelpers.ts

```diff
@@ -0,0 +1,10 @@
+/** Wraps a template function and escapes given string arguments. */
+export function createTemplate<T extends (...args: string[]) => string>(
```

Comment:
While we're going for string escaping, I thought it might be nice to switch to something more type-safe and use functions + string templates, would help keep track of variables used better. 

I could convert other templates at a later time also, but this could be a nice starting point, unless there's benefits to replacing that I might be overseeing.

In case we end up with content that needs to not be escaped somehow, we could expand this function to support more complex cases but for now I just wanted to create a wrapper.

Reply:
Would it make sense to  use handlebars or a similar templating engine rather than roll something ourselves?

Reply:
that might be nice though not sure how complex our templates can get; in any case I'd maybe personally prefer an engine written with TypeScript in mind like [eta](https://www.npmjs.com/package/eta)

With stuff like replacing characters we'd probably end up handling it ourselves or use a plugin with these template engines. Honestly to me having a function build a string seems like a nicer experience mainly because of arguments being exposed in the IDE though perhaps a template engine would be more scalable for the future.

cc: @alenakhineika

Reply:
With Gaurab we have discussed a revamp of how we do templates with things like adding shortcut information to them, it might be a better moment to discuss introducing i.e. a templating engine, I created [VSCODE-661](https://jira.mongodb.org/browse/VSCODE-661) to keep track of that.

I could go ahead with this setup for now just to have it in for release and we can change it to whatever we decide more globally later.